examples/advanced/experiment-tracking/tensorboard/README.md

```diff
@@ -61,32 +89,80 @@ $ tree /tmp/nvflare/jobs/workdir/server/simulate_job/
 ```
 
 
-### 5. Tensorboard Streaming
+### 5. View TensorBoard Results
+
+To view training metrics that are being streamed to the server, run:
 
-On the client side:
 
+```bash
+tensorboard --logdir=/tmp/nvflare/jobs/workdir/server/simulate_job/tb_events
 ```
-from nvflare.client.tracking import SummaryWriter
+
+Then open your browser to `http://localhost:6006` to view the metrics.
+
+**Note**: If the server is running on a remote machine, use port forwarding:
+```bash
+ssh -L 6006:127.0.0.1:6006 user@server_ip
 ```
-Instead of writing to TB files, this actually generates NVIDIA FLARE events of type `analytix_log_stats`.
-The `ConvertToFedEvent` widget will turn the event `analytix_log_stats` into a fed event `fed.analytix_log_stats`,
-which will be delivered to the server side.
 
-On the server side, the `TBAnalyticsReceiver` is configured to process `fed.analytix_log_stats` events,
-which writes the received TB data into appropriate TB files on the server.
+---
 
-To view training metrics that are being streamed to the server, run:
+## How It Works
 
+### Client-Side Tracking
 
-```
-tensorboard --logdir=/tmp/nvflare/jobs/workdir/server/simulate_job/tb_events
-```
+In `train_script.py`, the client code uses the NVFlare tracking API:
```

Comment:
we need to be clearer on "Client-Side" Tracking

there is "client.py" which is the user's own training script that needs to write a metric out.

there could be Receiver on the "NVFlare Client" side to receive the metrics.

there could be Receiver on the "NVFlare Server" side to receive the metrics.